### DISCUSSION TITLE
Any way to access custom garnet object inside custom transaction function?

### DISCUSSION THREAD

**Discussion by ShaolanTech** [microsoft/garnet#336] [General]
In CustomTransactionProcedure.Main, I found the member "RMW_ObjectStore" of IGarnetAPI, but there is no description in the office document, and the demos neither. For example,i need set a value to "MyDict" inside a custom transaction procedure.

**Comment by ShaolanTech** [comment]
ok, after few hours of digging into the code, i've got the answer and need to implement the resp protocol myself. hope for the detailed docs, it will save a lot of time for developers. Thanks!

**Comment by badrishc** [comment]
Thanks for the investigation. We could make this simpler in future, to allow users in a transactional procedure to access custom objects. Feel free to contribute in this space, thanks.
